ts/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@chainlink/solana-sdk",
+  "version": "0.2.0",
+  "repository": "https://github.com/smartcontractkit/chainlink-solana",
+  "license": "MIT",
+  "engines": {
+    "node": ">= 14"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "files": [
+    "dist/**/*",
+    "src",
+    "LICENSE",
+    "README.md"
+  ],
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "lint": "tsc --no-emit",
+    "test": "jest --runInBand",
+    "lint:format": "prettier --check ./src",
+    "build": "tsc",
+    "clean": "rm -rf dist",
+    "format": "prettier --write ./src"
+  },
+  "dependencies": {
+    "@project-serum/anchor": "^0.22.0",
+    "@solana/web3.js": "1.30.2"
```

Comment:
Can you bump the versions here? We recently bumped these on master to `0.23.0` and `1.31.0`